scripts/predicting_the_past_import.py

```diff
@@ -45,14 +45,16 @@ def import_pris(pris_link):
                        delimiter=',',
                        encoding='iso-8859-1',
                        skiprows=20,
-                       usecols=[2, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16]
                        )
+
+    pris = pris.rename(columns={pris.columns[2]: 'Country'})
+    pris = pris[['Country', 'Unit', 'Current Status', 'Type',
```

Comment:
I don't know what the PRIS dataset is nor what it looks like... What is "Country" before you rename it? This is better, for sure, but it still is not future-proof (``pris.columns[2]`` could end up being a different number in the future). 

I think it's better to continue using ``usecols`` and then if you need to rename a column you can still do that. That way someone else familiar with the dataset can see immediately which columns you're using rather than having to refer back to the dataset. But again, this is my opinion.

Reply:
PRIS is a database of all nuclear power plants in the world, maintained by the IAEA. The 'Country' column before it is renamed is the first (alphabetically) country that has nuclear plants, in this case Argentina. The alternative is ``pris.rename(columns={'ARGENTINA':'Country'})``. So this is me anticipating that the first country listed with nuclear plants changes before the IAEA changes how they format the data.